website/docs/tutorial/1-setting-up.md

```diff
@@ -121,3 +82,7 @@ There’s two main ways of working with dbt:
 To use the CLI, it's important that you know some basics of your terminal. In particular, you should understand `cd`, `ls` and `pwd` to navigate through the directory structure of your computer easily. As such, if you are new to programming, we recommend using **dbt Cloud** for this tutorial.
 
 If you wish to use the CLI, please follow the [installation instructions](/dbt-cli/install/overview) for your operating system.
+
+## Setting up your data platform
+
+Now that you have some context for this tutorial, let's get your environment set up.  Check out the table of contents to the left to set up your environment on BigQuery, Snowflake, or Redshift.  These guides are helpful for anyone with an existing data platform or starting from scratch.
```

Comment:
suggestion: Could we put some bullets here with references to the options?

Reply:
good call - Leona broke this section onto a different page and breaks down the options (on a different branch) so I think she captured that over there.